draft-ietf-scitt-architecture.md

```diff
@@ -713,12 +713,7 @@ Such policies may use as input all information in the Envelope, the Receipt, and
 
 Verifiers SHOULD offer options to store or share Receipts in case they are needed to audit the Transparency Services in case of a dispute.
 
-# Federation[^1]
-
-[^1]: This section needs work.
-{: source="Henk"}
-
-Editor's note: This section needs work.
+# Federation
```

Comment:
I would prefer adding a line, that says this item is open for discussion and point to a github issue where it is been tracked. We have done this for CoRIM. Please see https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html

Reply:
@yogeshbdeshpande could you please provide a suggestion for the meeting on Monday?